nixos/maintainers/scripts/ec2/amazon-image.nix

```diff
@@ -43,7 +43,7 @@ in {
 
     sizeMB = mkOption {
       type = with types; either (enum [ "auto" ]) int;
-      default = 2048;
+      default = 2252;
```

Comment:
The commit does not say why the disk size was bumped.

Reply:
Indeed, the Amazon test was broken due to not enough disk space, I don't know if it was induced by me or in-between latest release and master. Will double check.